gfx/GiantDays/tile_config.json

```diff
@@ -8493,7 +8493,7 @@
             "cookbook_sushi",
             "holybook_granth",
             "holybook_scientology",
-            "robofac_yrax_deactivation_manual"
```

Comment:
You should not expand the tile_config.json files manually. 

I'm pretty sure these will just get undone or cause conflicts in the next automated update for the tilesets.